integration_tests/utils/grpc_rest.go

```diff
@@ -54,8 +59,9 @@ type TestSuite struct {
 func (suite *TestSuite) GetGRPCConn() *grpc.ClientConn {
 	// Create a connection to the gRPC server.
 	grpcConn, err := grpc.Dial(
-		"127.0.0.1:26630",   // Or your gRPC server address.
-		grpc.WithInsecure(), // The SDK doesn't support any transport security mechanism.
+		"127.0.0.1:26630", // Or your gRPC server address.
```

Comment:
Is it a change in how to communicate with GRPC?
I think it would be great to have a file with all changes (in API, usage, breaking changes, etc.) after migration to the new version of cosmos. 
We will later include it also into the release notes.

Reply:
Yes, I made changes by following upgrade guideline
Agree, seems we should add CHANGELOG.md file